src/Sentry.Unity.Android/JniExecutor.cs

```diff
@@ -9,7 +9,7 @@ namespace Sentry.Unity.Android;
 internal class JniExecutor : IJniExecutor
 {
     // We're capping out at 16ms - 1 frame at 60 frames per second
-    private static readonly TimeSpan DefaultTimeout = TimeSpan.FromMilliseconds(16);
+    private static readonly TimeSpan DefaultTimeout = TimeSpan.FromMilliseconds(2000);
```

Comment:
What is this timeout used for? The comment above indicates some relationship to giving up after causing 1 frame dropped. 2 seconds is quite large of a UI freeze